src/DayColumn.js

```diff
@@ -174,9 +178,8 @@ class DayColumn extends React.Component {
     )
   }
 
-  renderEvents = () => {
+  renderEvents = ({ events, isBackgroundEvent }) => {
```

Comment:
Observation/Suggestion: It feels like the `isBackgroundEvent` property here may be confusing since the name is singular, but it applies to all events passed in.

How do we feel about keeping the current state where `renderEvents` doesn't accept arguments, and instead we get both `events` and `backgroundEvents` from the props within `renderEvents` and merge them into a single collection of events such that background events have a property `isBackgroundEvent` set to true?

Reply:
Merging `events` and `backgroundEvents` will cause [`no-overlap`](https://github.com/kamildebicki/react-big-calendar/blob/841a43612e4e108cd5a33bcb6dbcd0e9cd46aac7/src/utils/layout-algorithms/no-overlap.js) algorithm to treat them as colliding events, and place them next to each other instead of one on top of each other.